src/foreman_mcp_server/middleware/auth.py

```diff
@@ -1,14 +1,26 @@
+from pathlib import Path
+
 import apypie
 from fastmcp.server.middleware import Middleware, MiddlewareContext
 
 
 class AuthMiddleware(Middleware):
-    def __init__(self, foreman_url: str, verify_ssl: bool = True):
+    def __init__(
+        self, foreman_url: str, verify_ssl: bool = True, ca_bundle: str = None
+    ):
         # TODO: user_map can grow indefinitely, consider using a more efficient structure or cleanup mechanism
         self.user_map = {}
         self.foreman_url = foreman_url
         self.verify_ssl = verify_ssl
 
+        # Use default CA bundle location if none specified
+        if ca_bundle is None and verify_ssl:
+            default_ca_path = Path.cwd() / "ca.pem"
+            if default_ca_path.exists():
+                ca_bundle = str(default_ca_path)
```

Comment:
I don't think we need this. The `ca_bundle` param description says if it's not provided then the default system bundle will be used, but we then override it via `./ca.pem`, thus ignoring whatever the underlying library uses. In other words, we require `./ca.pem` to be present, but don't mention this.

I'd vote for dropping both these blocks.